lucetc/tests/wasm.rs

```diff
@@ -32,14 +52,14 @@ pub fn test_bindings() -> Bindings {
 
 mod module_data {
     /// Tests of the `ModuleData` generated by the lucetc Compiler
-    use super::load_wat_module;
+    use super::{load_wat_module, LoadOptions};
     use lucet_module::bindings::Bindings;
     use lucetc::{Compiler, CpuFeatures, HeapSettings, LucetcErrorKind, OptLevel};
     use std::path::PathBuf;
 
     #[test]
     fn exported_import() {
-        let m = load_wat_module("exported_import");
+        let m = load_wat_module("exported_import", LoadOptions::default());
```

Comment:
The rest of the changes in this function are noise from the new `load_wat_module` parameter, luckily via `Default` we don't need to be concerned with setting fields. :slightly_smiling_face: